nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -200,13 +268,19 @@ in
         group = cfg.group;
```

Comment:
Just to question, do we lose anything here by using DynamicUser ? You don't have to do that in this PR of course, but I was wondering if we would lose anything by not defining the user entirely.

Reply:
I was running `caddy` as `DynamicUser` for a while with success. We might need to consider a few special cases, but it should *probably* work.